libs/wingsdk/src/target-tf-aws/function.ts

```diff
@@ -213,7 +213,14 @@ export class Function extends cloud.Function implements IAwsFunction {
         }),
       },
       environment: {
-        variables: Lazy.anyValue({ produce: () => this.env }) as any,
+        variables: Lazy.anyValue({
+          produce: () => ({
+            ...this.env,
+            // enable source maps
+            NODE_OPTIONS:
+              (this.env.NODE_OPTIONS ?? "") + "--enable-source-maps",
+          }),
+        }) as any,
```

Comment:
WRT @hasanaburayyan's comments about perf, I would be fine removing this to work on an alternate solution later. The impact of `--enable-source-maps` is not negligible, especially with aws-sdk-sized sourcemaps.

I think the ideal scenario is that functions should make the sourcemap available as a preflight value, and the CLI and/or TestRunner can be responsible for pulling in sourcemaps as needed. This also makes sense if you want to export your sourcemaps for other usecases (like uploading to your error tracking system).

Reply:
With this flag, when V8 generates a stacktrace it automatically takes care of remapping the file positions based on the available sourcemaps.